common/app/model/content/Atoms.scala

```diff
@@ -127,6 +135,21 @@ object MediaAtom extends common.Logging {
       mimeType = mediaAsset.mimeType)
   }
 
+  def mediaImageAssetMake(mediaImage: AtomApiImageAsset, caption: String): ImageAsset = {
+    ImageAsset(
+      index = 0,
```

Comment:
Not sure what index is for... From what I can tell it was used for our lightbox implementation. Best to ask @rich-nguyen as he would know best.